src/rippled/lib/rippled.js

```diff
@@ -289,6 +289,20 @@ const getAccountNFTs = (rippledSocket, account, marker = '', limit = 20) =>
     limit,
   })
 
+const getNFTInfo = (rippledSocket, tokenId) =>
```

Comment:
this argument is not correctly named, since you cannot call this API against `rippled` but only `clio`

Reply:
it is still accepting a rippledSocket tho. it is only inside `queryP2P` where it checks for clio/p2p socket